nbconvert/exporters/tests/test_webpdf.py

```diff
@@ -6,7 +6,7 @@
 import io
 import pytest
 
-from mock import patch
```

Comment:
If we got all of the `from mock imports` changed we could also remove it from the requirements